codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -20,17 +21,66 @@ enum NotaryStatus {
 
 /// Visit a [Directory] type while examining the file system extracted from an
 /// artifact.
-Future<void> visitDirectory(Directory directory, String entitlementParentPath, Logger logger) async {
+Future<void> visitDirectory(Directory directory, String entitlementParentPath, Directory tempDir, Logger logger,
```

Comment:
For methods taking several parameters, prefer using named parameters. It's hard to remember the order, and makes it more obvious the intent.

Reply:
True, cut down to 2 parameters and changed to named parameters.